app/src/main/res/layout/fragment_address_editor.xml

```diff
@@ -324,6 +324,41 @@
 
         </com.google.android.material.textfield.TextInputLayout>
 
+        <!-- Country -->
+        <LinearLayout
+            android:id="@+id/country_layout"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="10dp"
+            android:paddingStart="3dp"
+            android:paddingEnd="0dp"
+            android:orientation="vertical"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/zip_layout"
+            app:layout_constraintBottom_toTopOf="@id/phone_title">
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textColor="?attr/textPrimary"
+                android:textSize="12sp"
+                android:labelFor="@id/country_drop_down"
+                app:fontFamily="@font/metropolis_semibold"
+                android:text="@string/addresses_country" />
+
+            <androidx.appcompat.widget.AppCompatSpinner
+                android:id="@+id/country_drop_down"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content" />
+
+            <View
+                android:layout_width="match_parent"
+                android:layout_height="1dp"
+                android:background="?attr/textPrimary" />
```

Comment:
is this supposed to be an underline? Would border primary be a better choice?

Reply:
I just borrowed this directly from the credit card editor. The text is intended to be black in light mode (for the current design), so `formDefault` is what I think we're looking for. I'll update that

Reply:
Ah looks like `formDefault` isn't available as an `?attr` so it might be simplest to leave as is until the compose migration.

Reply:
This is a `View` not a `TextView`. I'm fine with leaving it be though with the Compose migration on the roadmap.